docker/ds/README.md

```diff
@@ -44,5 +44,5 @@ for example, will not report the correct status.
 The directories internal metrics (exposed via Prometheus) report accurate status and can be used for monitoring and alerting. In addition
 the directory replication logs (logs/replication) show replication activity.
 
-These limitations will be removed in the 6.5/7.0 time frame.
+These limitations will be removed in the 7.0 time frame.
```

Comment:
You could remove this if you want, not sure what the benefit of mentioning it is.